src/ComposerRequireChecker/DependencyGuesser/GuessFromInstalledComposerPackages.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ComposerRequireChecker\DependencyGuesser;
+
+use Roave\BetterReflection\BetterReflection;
+use Roave\BetterReflection\Identifier\Identifier;
+use Roave\BetterReflection\Identifier\IdentifierType;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflector\DefaultReflector;
+use Roave\BetterReflection\SourceLocator\Type\Composer\Factory\MakeLocatorForInstalledJson;
+use Generator;
+use Roave\BetterReflection\SourceLocator\Type\MemoizingSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\SourceLocator;
+use function preg_match;
+use function preg_quote;
+use function sprintf;
+use function str_replace;
+use const DIRECTORY_SEPARATOR;
+
+final class GuessFromInstalledComposerPackages implements Guesser
+{
+    private SourceLocator $sourceLocator;
+    private string $pathRegex;
+
+    public function __construct(string $installationPath)
+    {
+        $this->sourceLocator = new MemoizingSourceLocator((new MakeLocatorForInstalledJson())(
```

Comment:
This locator does not include PHP symbols

Reply:
I just realized that this is potentially also broken in `roave/backward-compatibility-check` :|

Reply:
Not quite sure but dont we have for the php internal stuff the other guesser?

Reply:
Possibly, but BetterReflection can also reflect all of that, heh :D

Reply:
I would say, follow up PR, if wanted.